core/cmd/remote_client.go

```diff
@@ -753,8 +753,41 @@ func (cli *Client) SetLogSQL(c *clipkg.Context) (err error) {
 		}
 	}()
 
-	var lR webPresenter.LogResource
-	err = cli.renderAPIResponse(resp, &lR)
+	var svcLogConfig webPresenter.ServiceLogConfigResource
+	err = cli.renderAPIResponse(resp, &svcLogConfig)
+	return err
+}
+
+// SetLogPkg sets the package log filter on the node
+func (cli *Client) SetLogPkg(c *clipkg.Context) (err error) {
+	pkg := strings.Split(c.String("pkg"), ",")
+	level := strings.Split(c.String("level"), ",")
+
+	serviceLogLevel := make([][2]string, len(pkg))
+	for i, p := range pkg {
+		serviceLogLevel[i][0] = p
+		serviceLogLevel[i][1] = level[i]
+	}
+
+	request := web.LogPatchRequest{ServiceLogLevel: serviceLogLevel}
```

Comment:
This is going to clobber the database every time, no? That's quite confusing. If the nop sets a log level via web UI, will this flag overwrite it on reboot?

Reply:
This CLI command will allow for setting the service level logs to be persisted in the db, the web UI would call the same `/v2/log` endpoint so it should do the same thing

On reboot we don't have env vars for service level logging so it should always get the state from the db, and default to `info` if no log level is set ([logger.go ln 176](https://github.com/smartcontractkit/chainlink/pull/4102/files#diff-a64e2e2ab03a93571b59b325a8acadd205a5964c89f2bbb63eabaf118d544440R176))

Reply:
Perhaps I have misunderstood, does this line not set the logging level in the DB from the command line?

https://github.com/smartcontractkit/chainlink/pull/4102/files#diff-df09f6f4f417bf267523b513ebd19590cf8d4c1587510aa3f4682d5749c91feaR161

Reply:
Ah I see what you're saying, I interpreted it's use as one time cli commands like: 

`./chainlink config logpkg --pkg head_tracker --level debug`

It seems that other commands in this section are used this way but that could be an incorrect assumption. If the nop set's these on start of the app then yes it will override and hit db...is there a way we should limit this?